e2e/driver/CheReporter.ts

```diff
@@ -153,6 +155,15 @@ class CheReporter extends mocha.reporters.Spec {
       const browserLogsStream = fs.createWriteStream(browserLogsFileName);
       browserLogsStream.write(new Buffer(browserLogs));
       browserLogsStream.end();
+
+      // stop and remove running workspace
+      if (TestConstants.DELETE_WORKSPACE_ON_FAILED_TEST) {
+        console.log("Property DELETE_WORKSPACE_ON_FAILED_TEST se to true - trying to stop and delete running workspace.")
```

Comment:
This check
```
if (TestConstants.DELETE_WORKSPACE_ON_FAILED_TEST)
```
means "true" if constant is **not empty**,  **not undefined** and **not null**
but in case with ```TestConstants.DELETE_WORKSPACE_ON_FAILED_TEST``` it will not work
because you initialize this constant by string value by default.
```
    DELETE_WORKSPACE_ON_FAILED_TEST: process.env.DELETE_WORKSPACE_ON_FAILED_TEST || 'false'
```
this "if" block will trigered just in case if someone put to the ```DELETE_WORKSPACE_ON_FAILED_TEST``` an empty value.

I suppose that you wanted to create boolean constant
So, in this case, you should change 
```
    DELETE_WORKSPACE_ON_FAILED_TEST: process.env.DELETE_WORKSPACE_ON_FAILED_TEST === 'true'
```
in this case it will be a boolean constant with "false" value by default and "if" block will work correctly.

Reply:
Good catch, I've changed that as you suggested.